src/api/helpers/map-CIS-countries/map-CIS-country/no-insurance-support/a-and-b-rating-conditions copy/index.ts

```diff
@@ -0,0 +1,52 @@
+import { EXTERNAL_API_DEFINITIONS } from '../../../../../constants';
```

Comment:
are there any tests for this file?

Reply:
Whoops, this was left over and not required. Have deleted